app/src/test/java/org/simple/clinic/registration/pin/RegistrationPinScreenControllerTest.kt

```diff
@@ -126,6 +126,7 @@ class RegistrationPinScreenControllerTest {
   }
 
   private fun setupController(
+      requiredPinLength: Int = 4,
```

Comment:
Can we use the global constant `SECURITY_PIN_LENGTH` here?

Reply:
Good catch, done!